src/up.cmd.js

```diff
@@ -208,7 +241,7 @@ export default class UpCommand extends AbstractCommand {
     this.emit('started', this);
     if (this._open) {
       const url = this._open.startsWith('/')
-        ? `http://localhost:${this._project.server.port}${this._open}`
+        ? `${this._tls ? 'https' : 'http'}://localhost:${this._project.server.port}${this._open}`
```

Comment:
I would prefer to set the `scheme` correctly somewhere instead to infer it from the `_tls` everywhere,

Reply:
I agree.  This isn't ideal.  At the same time, I wanted to mostly just support the tls feature without having to make people understand they have to set a myriad of other settings (like forgetting to set the "scheme" feature)...

Reply:
Okay, I see your point and I changed my code to refer to `this._scheme` which will be modified in the case of using tls.